sfm_pc/management/commands/import_google_doc.py

```diff
@@ -1733,7 +1733,7 @@ def create_person(self, person_data):
                     role = None
                 else:
                     if role_name:
```

Comment:
if role_name can have whitespace in it, seems safest to do the strip *before* evaluating the truthyness of role_name.

Reply:
Great call, @fgregg, done for both role and rank in https://github.com/security-force-monitor/sfm-cms/pull/775/commits/4a370572e03c921e4329c14951aebd71b4f2ba4e.